mne/beamformer/_dics.py

```diff
@@ -190,12 +612,15 @@ def dics_epochs(epochs, forward, noise_csd, data_csd, reg=0.05, label=None,
     Compute a Dynamic Imaging of Coherent Sources (DICS) beamformer
     on single trial data and return estimates of source time courses.
 
+    .. warning:: This function is deprecated and will be removed in MNE 0.16.
+                 Use make_dics/apply_dics_* instead.
```

Comment:
use the `@deprecated` decorator from `mne.utils` instead, I think it modifies docstring and automatically emits a warning on use